ArduPlane/config.h

```diff
@@ -340,9 +340,14 @@
 #define PARACHUTE ENABLED
 #endif
 
+//////////////////////////////////////////////////////////////////////////////
+// Payload Gripper
+#if !HAL_MINIMIZE_FEATURES && !defined(GRIPPER_ENABLED)
```

Comment:
This should still be the ifndef first, and then inside there we should do the HAL_MINIMIZE_FEATURES check. And use that to define GRIPPER_ENABLED as either ENABLED or DISABLED. (Otherwise we generate a bunch of unneeded warnings on PX4-v2 builds)

Reply:
Fixed